tests/scripts/all.sh

```diff
@@ -1197,6 +1197,33 @@ component_test_psa_external_rng_no_drbg_use_psa () {
     tests/ssl-opt.sh -f 'Default\|opaque'
 }
 
+component_test_full_no_cipher () {
+    msg "build: full minus CIPHER"
+    scripts/config.py full
+    scripts/config.py unset MBEDTLS_CIPHER_C
```

Comment:
As for MD, please add comments for direct/indirect dependencies and sort alphabetically within each category.